src/main/java/edu/wpi/grip/ui/util/CSSElementUtility.java

```diff
@@ -0,0 +1,59 @@
+package edu.wpi.grip.ui.util;
+
+import edu.wpi.grip.core.Connection;
+import edu.wpi.grip.core.Operation;
+import edu.wpi.grip.core.Step;
+
+public final class CSSElementUtility {
+
+    private CSSElementUtility(){}
+
+
+    /**
+     * Returns the CSS id name for an operation
+     * @param operation The operation to get the class name for
+     * @return The CSS id for the operation. To use as a css selector then prepend the string with a '#'
+     */
+    public static String idNameFor(Operation operation){
```

Comment:
ID vs Class
The id is assigned as the elements ID param instead of assigning it as a class on the element.